src/clients/db/AppointmentsRepository.ts

```diff
@@ -6,6 +6,38 @@ export class AppointmentRepository extends Repository<Appointment> {
   constructor() {
     super(appointments)
   }
+
+  // Fuzzy search for patient appointments. Used for patient appointment search bar
+  async searchPatientAppointments(patientId: string, text: string): Promise<Appointment[]> {
```

Comment:
Searching this list is not something that I had previously considered. I think that this is a really neat idea!

@fox1t @tehKapa @lauggh @StefanoMiC, 

What does everyone thing the requirements should be here? I see a really cool opportunity for having filter buttons for the different appointment types.

Reply:
We can leave it like this for now. We need to open a new feature issue to search about this. We really want a multi-field fuzzy search on almost every entity.